docs/language/functions/map.md

```diff
@@ -0,0 +1,42 @@
+### Function
+
+&emsp; **map** &mdash; apply a function to each element of an array or set
+
+### Synopsis
+
+```
+map(v: array|set, f: function) -> array|set
+```
+
+### Description
+
+The _map_ function applies function f to every element in array/set v and
+returns the augmented array/set. Function f must be a function that takes
```

Comment:
Nits: "or" is classier than "/". And "augmented" suggests that values returned by f are added to v.
```suggestion
The _map_ function applies function f to every element in array or set v and
returns an array or set of the results. Function f must be a function that takes
```